auditbeat/module/auditd/ecs_categorization.yml

```diff
@@ -0,0 +1,320 @@
+# Useful links:
+# https://raw.githubusercontent.com/torvalds/linux/v4.16/include/uapi/linux/audit.h
+# https://raw.githubusercontent.com/linux-audit/audit-userspace/4d933301b1835cafa08b9e9ef705c8fb6c96cb62/lib/libaudit.h
+# https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html
+# https://github.com/elastic/go-libaudit/blob/master/aucoalesce/event_type.go
+# https://github.com/elastic/go-libaudit/blob/master/aucoalesce/normalizations.yaml
+
+eventTypes:
+  EventTypeUserAccount:
+    # AUDIT_ADD_USER - User account added
+    # AUDIT_DEL_USER - User account deleted
+    # AUDIT_ADD_GROUP - Group account added
+    # AUDIT_DEL_GROUP - Group account deleted
+    # AUDIT_GRP_MGMT - Group account attr was modified
+    # AUDIT_GRP_CHAUTHTOK - Group acct password or pin changed
+    # AUDIT_ACCT_LOCK - User's account locked by admin
+    # AUDIT_ACCT_UNLOCK - User's account unlocked by admin
+    categories:
+      - iam
+    default_types:
+      - info
+    types:
+      AUDIT_ADD_USER:
+        - user
+        - creation
+      AUDIT_USER_MGMT:
+        - user
+        - change
+      AUDIT_DEL_USER:
+        - user
+        - deletion
+      AUDIT_GRP_MGMT:
+        - group
+        - change
+      AUDIT_GRP_CHAUTHTOK:
+        - group
+        - change
+      AUDIT_ADD_GROUP:
+        - group
+        - creation
+      AUDIT_DEL_GROUP:
+        - group
+        - deletion
+
+  EventTypeUserLogin:
+    # AUDIT_USER_AUTH - User system access authentication
+    # AUDIT_USER_ACCT - User system access authorization
+    # AUDIT_USER_MGMT - User acct attribute change
+    # AUDIT_CRED_ACQ - User credential acquired
+    # AUDIT_CRED_DISP - User credential disposed
+    # AUDIT_USER_START - User session start
+    # AUDIT_USER_END - User session end
```

Comment:
Been a long time since we worked on this. But I remember we wanted to treat sessions and authorizations differently than authentications. In other words, there's no category published for it yet.

Am I remembering this right, @dainperkins and @MikePaquette?